radixdlt/src/main/java/com/radixdlt/statecomputer/RadixEngineStateComputer.java

```diff
@@ -298,4 +310,21 @@ public void commit(VerifiedCommandsAndProof verifiedCommandsAndProof) {
 			}
 		}
 	}
+
+	@Override
+	public void commit(VerifiedCommandsAndProof verifiedCommandsAndProof, VerifiedVertexStoreState vertexStoreState) {
+		atomicCommitManager.startTransaction();
+		try {
+			commitInternal(verifiedCommandsAndProof);
```

Comment:
I'm assuming here that commitInternal uses the open transaction, despite it not being mentioned here.